projects/manage-supervision-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/overview/entity/Event.kt

```diff
@@ -59,7 +59,17 @@ class Event(
 
     @Column(name = "created_datetime")
     val dateCreated: ZonedDateTime,
-)
+) {
+    fun isInactiveEvent(): Boolean {
+        if (disposal != null && disposal.active.not())
+            return true
+
+        if (active.not())
+            return true
+
+        return false
+    }
```

Comment:
It might be me but this is quite hard to get my head around, probably just because it's a double negative.  Maybe we could simplify it?

E.g. to

```suggestion
    fun isActiveEvent() = active && (disposal == null || disposal.active)
```

or 

```suggestion
    fun isInactiveEvent() = !active || (disposal?.active == false)
```

Reply:
i did have two methods, but the logic i needed into one, as i found them easier to read